lib/trento_web/openapi/schema/provider.ex

```diff
@@ -15,6 +15,18 @@ defmodule TrentoWeb.OpenApi.Schema.Provider do
     })
   end
 
+  defmodule FilterableProviders do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "FilterableProvider",
+      type: :string,
+      description: "A provider that can be used to filter the Catalog",
+      # default?
```

Comment:
I would personally include `default` here, as we actually have the data, but I understand that is not really meaningful.

Reply:
Fine to me, adding it.